packages/kbn-dev-utils/src/run/metrics.ts

```diff
@@ -23,39 +23,43 @@ export class Metrics {
     this.reporter = CiStatsReporter.fromEnv(log);
     this.meta = new Map();
     this.startTime = Date.now();
-    this.filePath = path.relative(REPO_ROOT, process.argv[1]).replace('.js', '');
+
+    // standardize to unix path
+    this.filePath = path
+      .relative(REPO_ROOT, process.argv[1])
+      .replace('.js', '')
+      .split(path.sep)
+      .join('/');
```

Comment:
Mind if we go with `normalize-path` here instead?

Reply:
f5bb7feeee57e20b8ebed400d4815e21410237b3